app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt

```diff
@@ -193,7 +193,7 @@ class EnrollmentActivity : ActivityGlobalAbstract(), EnrollmentView {
 
     override fun openEvent(eventUid: String) {
         if (presenter.isEventScheduleOrSkipped(eventUid)) {
-            val scheduleEventIntent = ScheduledEventActivity.getIntent(this, eventUid)
+            val scheduleEventIntent = ScheduledEventActivity.getIntent(this, eventUid, "", "")
```

Comment:
why in enrollment activity is a empty teiUid and enrolmmentUid?

Reply:
I added optional field teiUid and enrollmentUid which can have default value of empty strings, now I have moved the default value for those fields in the definition of the function itself and no need to pass empty value when calling the function so I fixed that in the next commit.